guider.h

```diff
@@ -422,11 +426,15 @@ inline double Guider::GetMinStarHFDDefault() const
     return 1.5;
 }
 
-inline double Guider::getMinStarSNR() const
+inline double Guider::GetAFMinStarSNR() const
 {
-    return m_minStarSNR;
+    return m_minAFStarSNR;
 }
 
+inline double Guider::GetMaxStarHFD() const
+{
+    return m_maxAFStarHFD;
+}

```

Comment:
cosmetic, match formatting of surrounding code
```suggestion
}

```